packages/documentation/cypress/e2e/components/card.cy.ts

```diff
@@ -12,7 +12,7 @@ describe('Card', () => {
           'heading-order': {
             enabled: false,
           },
-          'aria-allowed-attr': {
+          'aria-prohibited-attr': {
```

Comment:
@alizedebray e2e are failing because of cypress-axe. Axe-core has been updated from your dependencies (for some reason) to 4.8.4 and now make a difference between "aria-allowed-attr" and "aria-prohibited-attr" rules.

Reply:
I will have a look